shared/libraries/opcuatms/opcuatms_client/src/objects/tms_client_component_impl.cpp

```diff
@@ -144,12 +169,17 @@ ErrCode TmsClientComponentBaseImpl<Impl>::setVisible(Bool visible)
     }
     catch (...)
     {
-        const auto loggerComponent = this->daqContext.getLogger().getOrAddComponent("OpcUaClient");
-        LOG_D("OPC UA Component {} failed to set \"Active\" state.", this->localId);
+        const auto loggerComponent = getLogger();
```

Comment:
I'd rename `getLogger` to `getLoggerComponent`, and maybe add it to some `details::` namespace.